apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -1036,9 +1036,9 @@ static int _wm_test_join(struct options *opt, int argc, char *argv[])
 	}
 
 	if (opt->auth_type == WIFI_MANAGER_AUTH_WEP_SHARED) {
-		if (strlen(argv[5]) == 13) {
+		if ((strlen(argv[5]) == 13) || (strlen(argv[5]) == 26)) {
```

Comment:
How can we discriminate hex from ascii?

Reply:
You can discriminate it by its password length, 10 or 26 means HEX, 5 or 13 means ASCII

Reply:
How about 'asdfg12345'? Is it correct to consider this as a hex value?

Reply:
"asdfg12345" is invalid wep password, because hex password can be only from 0~9, a~f,A~F. But the _wm_test_join() here only determine whether its WIFI_MANAGER_CRYPTO_WEP_128 or WIFI_MANAGER_CRYPTO_WEP_64. So the modification is okay.